lib/showoff.rb

```diff
@@ -437,85 +435,170 @@ def process_markdown(name, section, content, opts={:static=>false, :pdf=>false,
       final
     end
 
+    def custom_syntax_templates 
+      {
+        "current" => {
+          "matchers" => [
```

Comment:
`"matchers"` and `"block"` should be symbols, `:matchers` and `:block`.

(In Ruby, symbols should be used the same way enums are used in C.)